### DISCUSSION TITLE
Difference between original_commit_id and commit_id of a review comment?

### DISCUSSION THREAD

**Discussion by amit1rrr** [community/community#24449] [Apps, API and Webhooks]
<p>What’s the difference between original_commit_id and commit_id of a review comment?</p>
<p>When would these be different? Documentation is not very helpful in explaining attributes.</p>
<pre><code>originalCommit: Identifies the original commit associated with the comment.

commit: Identifies the commit associated with the comment.
</code></pre>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/amit1rrr">@amit1rrr</a>.,</p>
<p>Thanks for reaching out! If there is a change to the PR: if a subsequent commit modifies the line you specify as the <code>position</code>. <code>original_commit_id</code>should refer to the previous commit id before the last change. I hope this helps!</p>

**Comment by amit1rrr** [comment]
<p>In the following PR, original_commit_id of one of the review comment is not even present in the list of PR commits. How can the commits disappear? Any way to fetch state of a file when that comment was made?</p>
<p>E.g. </p>
<p>PR: <a href="https://github.com/Microsoft/Recommenders/pull/681/" rel="nofollow noopener">https://github.com/Microsoft/Recommenders/pull/681/</a></p>
<p>Review comment: <a href="https://github.com/Microsoft/Recommenders/pull/681/#discussion_r269028725" rel="nofollow noopener">https://github.com/Microsoft/Recommenders/pull/681/#discussion_r269028725</a></p>
<p>This above review comment has original_commit_id as 168eea741e804980efe94e173088850966481d5b but this commit is not even present in the list of PR commits <a href="https://github.com/Microsoft/Recommenders/pull/681/commits" rel="nofollow noopener">here</a>. </p>
<p>Any ideas why would this be happening?</p>
